packages/xo-web/src/xo-app/vm-import/index.js

```diff
@@ -243,7 +243,7 @@ export default class Import extends Component {
 
   _import = () => {
     const { state } = this
-    return importVms(
+    const importVms1 = importVms(
```

Comment:
Why?

Reply:
probably a cleanup issue there must have been a console.log() here during the development